flow/flow.cpp

```diff
@@ -21,9 +21,24 @@
 #include "flow/flow.h"
 #include "flow/DeterministicRandom.h"
 #include "flow/UnitTest.h"
+#include "flow/rte_memcpy.h"
+#include "flow/folly_memcpy.h"
```

Comment:
I left it in because it's faster for short copies, which is the main use case for manually inlining memcpy calls.  Unfortunately, it's slower for long copies, which is the common "default" case that this PR targets.